src/OpenTracing/Span.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace OpenTracing;
+
+/**
+ * Span represents a unit of work executed on behalf of a trace. Examples of
+ * spans include a remote procedure call, or a in-process method call to a
+ * sub-component. A trace is required to have a single, top level "root"
+ * span, and zero or more children spans, which in turns can have their own
+ * children spans, thus forming a tree structure.
+ *
+ * @package OpenTracing
+ */
+interface Span {
+	/**
+	 * Sets or changes the operation name.
+	 *
+	 * @param string $operationName
+	 * @return $this
+	 */
+	public function setOperationName( $operationName );
+
+	/**
+	 * Indicates that the work represented by this span has been completed
+	 * or terminated, and is ready to be sent to the Reporter.
+	 *
+	 * If any tags / logs need to be added to the span, it should be done
+	 * before calling finish(), otherwise they may be ignored.
+	 *
+	 * @param int $finishTime
```

Comment:
Q: does php have a commonly used type for timestamps, like http://php.net/manual/en/class.datetimeinterface.php?  If it must be integer, we need to clarify the format of the value, e.g. microseconds since Unix epoch, etc.